test/transactions.go

```diff
@@ -32,3 +37,8 @@ func (t *transferTransaction) WithAmount(amount uint64) *transferTransaction {
 	t.builder.Transaction.InputArguments[0].Uint64Value = amount
 	return t
 }
+
+func (t *transferTransaction) WithInvalidContent() *transferTransaction {
```

Comment:
The 30 minutes constant should be configurable so let's use an over the top number in the meantime like a year

Reply:
Notice that it's 35 minutes into the future. A future timestamp is invalid as stated in spec:
https://github.com/orbs-network/orbs-spec/blob/master/behaviors/services/transaction-pool.md#addnewtransaction-method

Reply:
@OdedWx will add a new type to TransactionStatus to indicate this